packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1459,6 +1484,15 @@ class ListView extends BoxScrollView {
   /// the scroll position changes drastically.
   final double? itemExtent;
 
+  /// If non-null, forces the children to have the same extent as the given
+  /// widget in the scroll direction.
+  ///
+  /// Specifying an [prototypeItem] is more efficient than letting the children
```

Comment:
Add a comment (maybe in a "See also:" section) about the itemExtent property as an alternative.